src/containers/header/index.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+import CButton from 'src/components/CButton';
+import CLink from 'src/components/CLink';
+
+import LogoIcon from 'src/svgs/LogoIcon';
+import LogoTypography from 'src/svgs/LogoTypography';
+import RingBell from 'src/svgs/RingBell';
+import Stellar from 'public/images/Stellar.svg';
+
+const Header = () => {
+  return (
+    <header className="flex justify-between items-center py-3 px-6">
+      <div className="inline-flex gap-[14px]">
+        <LogoIcon />
+        <LogoTypography />
+      </div>
+      <div className="inline-flex gap-[12px] items-center md:hidden sm:hidden">
+        <span className="mr-3">
+          <CLink title="Documentation" url="/" />
```

Comment:
Create a file called externalPages in constants and add links just like Pages file in there and import from there.

So it should be something like ExternalPages.DOCUMENTATION